contributors/triage/guideline.md

```diff
@@ -0,0 +1,60 @@
+# Kubernetes Issue Triage Guidelines
+
+## Purpose
+
+Speed up issue management.
+
+The Kubernetes issues are listed at https://github.com/kubernetes/kubernetes/issues and are identified with labels. For example, an issue that belongs to SIG Network will eventually be set to label `sig/network`. New issues will start out without any labels. The detailed list of labels can be found at https://github.com/kubernetes/kubernetes/labels. While working on triaging issues you may not have privilege to assign specific label (e.g. `triaged`) and in that case simply add a comment in the issue with your findings.
```

Comment:
@grodrigues3 

Is labels.yaml documented anywhere?

https://github.com/kubernetes/test-infra/blob/6f7432c18267d6e9bfd6732d30e2ea4725326a99/mungegithub/mungers/check-labels.go#L42

Do we translate area labels to sig labels yet?

We should probably delete the `triaged` label. It was part of experiment to estimate how long it took to triage issues.